App/localization/fr/index.ts

```diff
@@ -0,0 +1,85 @@
+const translation = {
+  "Global": {
+    "enter_pin": "Entrez le code PIN",
```

Comment:
I'm not clear what the case convention is for i18n, but should we keep case consistent (i.e. either pascal, camel or snake). Just nitpicking here.

Reply:
I am referring this one https://lokalise.com/blog/translation-keys-naming-and-organizing/ as of now but if you have any suggestion please share. I have kept the page name as pascal and snake case for the text key.